test/roundrobin_test.go

```diff
@@ -173,7 +175,9 @@ func (s) TestRoundRobin_NewAddressWhileBlocking(t *testing.T) {
 
 	// Send a resolver update with a valid backend to push the channel to Ready
 	// and unblock the above RPC.
-	r.UpdateState(resolver.State{Addresses: []resolver.Address{{Addr: backends[0].Address}}})
+	r.UpdateState(resolver.State{Endpoints: []resolver.Endpoint{
```

Comment:
This needed to be done eventually.  But maybe for similar things in the future...it could have been easier for this PR (and may still be easier when dealing with future migration PRs) to make the manual resolver's `UpdateState` and `InitialState` methods set `Endpoints` according to the default behavior if it's unset.